main.go

```diff
@@ -208,10 +208,18 @@ func fetchOrCreateTestPlan(ctx context.Context, apiClient *api.Client, cfg confi
 
 	handleError := func(err error) (plan.TestPlan, error) {
 		if errors.Is(err, api.ErrRetryTimeout) {
-			fmt.Println("Could not fetch or create plan from server, using fallback mode. Your build may take longer than usual.")
+			fmt.Println("\033[33mCould not fetch or create plan from server, falling back to non-intelligent splitting. Your build may take longer than usual.\033[0m")
 			p := plan.CreateFallbackPlan(files, cfg.Parallelism)
 			return p, nil
 		}
+
+		if billingError := new(api.BillingError); errors.As(err, &billingError) {
+			fmt.Println("\033[33m" + billingError.Message)
+			fmt.Println("Falling back to non-intelligent splitting. Your build may take longer than usual.\033[0m")
```

Comment:
I made the text yellow and updated the message.
What do you think @niceking @wooly ?
I'm planning to refactor the logging using a logger package in the future to tidy up the logging, but I think this is fine for now.

Reply:
Yeah it's fine for now, I wonder how the yellow highlight looks in light mode on Buildkite? 
<img width="503" alt="Screenshot 2024-09-17 at 12 13 11 PM" src="https://github.com/user-attachments/assets/6cb655b3-5294-4174-893a-fdd15529da44">
Is it a similar yellow to this WARN here? Perhaps for accessibility/readability reasons, might be better use like the alert or danger emoji or similar?

Reply:
Yes, it will be that yellow color. I have updated the log to use the ⚠️ emoji instead.